src/Memcached.php

```diff
@@ -647,7 +647,10 @@ protected function getExceptionByResultCode($code)
                 );
 
             default:
-                return new Exception\RuntimeException($this->getMemcachedResource()->getResultMessage());
+                return new Exception\RuntimeException(
+                    $this->getMemcachedResource()->getResultMessage(),
+                    $code
```

Comment:
Overall OK, but needs a test verifying that this passthrough is preserved, in order to prevent regressions.